tests/wallet.rs

```diff
@@ -688,3 +688,21 @@ fn inscribe_gif() {
     ),
   )
 }
+
+#[test]
+fn wallet_balance() {
+  let rpc_server = test_bitcoincore_rpc::spawn_with(Network::Regtest, "ord");
+
+  CommandBuilder::new("--regtest wallet balance")
+    .rpc_server(&rpc_server)
+    .expected_stdout("0\n")
+    .run();
+
+  let coinbase_tx = &rpc_server.mine_blocks_with_subsidy(1, 1_000_000)[0].txdata[0];
+  let amount = coinbase_tx.output[0].value;
+
+  CommandBuilder::new("--regtest wallet balance")
+    .rpc_server(&rpc_server)
+    .expected_stdout(format!("{amount}\n"))
```

Comment:
I think I'd avoid the constant, since it requires an extra line and doesn't do a ton for readability:

```suggestion
  CommandBuilder::new("--regtest wallet balance")
    .rpc_server(&rpc_server)
    .expected_stdout("1000000\n")
```

You could also use mine_blocks and make the value 5000000000.